app/views/articles/index.html.erb

```diff
@@ -17,7 +17,7 @@
                               action: "keyup->refresh-list#update"
                             } %>
           <%= button_tag type: 'submit', class: "search-bar-submit", aria: { label: "submit-search" } do %>
-            <i class="fas fa-search"></i>
```

Comment:
Just double checking - did you make sure the other icons are still OK? We use them in the dashboard as well and they are the old version.